packages/fuels-contract/src/contract.rs

```diff
@@ -526,7 +621,6 @@ where
     /// your method returns `bool`, it will be a bool, works also for structs thanks to the
     /// `abigen!()`). The other field of CallResponse, `receipts`, contains the receipts of the
     /// transaction.
-    #[tracing::instrument]
```

Comment:
If this is not needed, can we remove `tracing` from the the Cargo file?

Reply:
Good catch! I will have a look.

Reply:
This feature is not working since we changed the test structure :/ https://github.com/FuelLabs/fuels-rs/pull/607
Since we have logging, (and now also logging for require ) I think we can remove it completely. If a user wants to print the receipts he can do it manually. What do you guys think ?